docs/changelog.md

```diff
@@ -0,0 +1,2 @@
+```{include} ../CHANGELOG.md
```

Comment:
This doesn't seem to be working, or I can't see it on the test site. I assume it's supposed to be generated on the fly?